### PR TITLE
Remove serialization attributes

### PR DESCRIPTION
**Pull request by moodmosaic**
This pull request removes the serialization attributes and adds basic support for error handling. Essentially it will be part of a series of small pull requests related to #5.

### CODE REVIEW

File: Src/Qaiain.CommandLine/Program.fs [hunk 1]
```diff
@@ -9,41 +9,54 @@ module AzureQ =
...
+        | EmailData of EmailData
+        | Unknown
+
+    open System.Xml
+
+    let private toEmailData (xml : XmlDocument) =
+        let ns = XmlNamespaceManager(xml.NameTable)
+        ns.AddNamespace("e", "urn:grean:schemas:email:2014")
+
+        let select xp = xml.DocumentElement.SelectSingleNode(xp, ns).InnerText
```
Comment (ploeh) [inline Src/Qaiain.CommandLine/Program.fs]: What does `xp` mean?

File: Src/Qaiain.CommandLine/Program.fs [hunk 2]
```diff
@@ -9,41 +9,54 @@ module AzureQ =
...
+        let selectAll xp = xml.DocumentElement.SelectNodes(xp, ns)
+
+        try
+            {
+                From = { SmtpAddress = select <| "e:from/e:smtp-address"
```
Comment (ploeh) [inline Src/Qaiain.CommandLine/Program.fs]: I'm wondering... do you really need that reverse pipe when the function only takes a single argument?

File: Src/Qaiain.CommandLine/Program.fs [hunk 3]
```diff
@@ -9,41 +9,54 @@ module AzureQ =
...
+                                        DisplayName = n.LastChild.InnerText } }
+                     |> Seq.toArray
+
+                Subject = select <| "e:subject"
+                Body = select <| "e:body"
+            }
+            |> EmailData
+        with | :? NullReferenceException -> Unknown
+
+    let Parse input =
```
Comment (ploeh) [inline Src/Qaiain.CommandLine/Program.fs]: Please rename to `parse`. Qaiain follows the de-facto naming convention for F#, which is to use camel-case for function names. I know that's not what the official documentation says, but it's what most F# code looks like - even code being developed today...

(F# code that should be exposed to other .NET languages should still follow the normal .NET coding conventions, but those mostly apply to classes and their members.)

File: Src/Qaiain.CommandLine/Program.fs (hunk 1 above)
Comment (moodmosaic) [inline Src/Qaiain.CommandLine/Program.fs]: XPath but agreed though, _xp_ should be renamed.

Comment (ploeh) [inline Src/Qaiain.CommandLine/Program.fs]: Ah, okay :) How about `path`?

Comment (moodmosaic) [inline Src/Qaiain.CommandLine/Program.fs]: Yeap, sounds good :)

File: Src/Qaiain.CommandLine/Program.fs (hunk 2 above)
Comment (moodmosaic) [inline Src/Qaiain.CommandLine/Program.fs]: OMG yes... I will fix that (the reverse pipe is a bit redundant there). Thank you!

File: Src/Qaiain.CommandLine/Program.fs (hunk 3 above)
Comment (moodmosaic) [inline Src/Qaiain.CommandLine/Program.fs]: OK, no problem.

**Comment by moodmosaic** [comment]
I added some commits to address the comments above.

**Comment by ploeh** [comment]
Thank you for your contribution!

**Comment by moodmosaic** [comment]
Thank _you_ for the help and the great feedback as always!

**Pull request merged**